docs/docs/using-semaphore/notifications.md

```diff
@@ -117,17 +115,16 @@ sem create notification <name> \
     --slack-endpoint <slack-webhook-endpoint>
 ```
 
-
 Use the [Slack webhook](https://slack.com/help/articles/360041352714-Build-a-workflow--Create-a-workflow-that-starts-outside-of-Slack) for your Slack Workspace
 
-For example, to create a notification on the "web", "cli", and "api" projects on the "master" branch:
-
+For example, to create a notification on the "web", "cli", and "api" projects on the "master" branch for tags ending with "-rc" (like "v1.0-rc")
```

Comment:
For example, to create a notification on the "web", "cli", and "api" projects for pipelines that run on the "master" branch, or on  tags ending with "-rc" (like "v1.0-rc")